projects/ngx-indexed-db/src/lib/ngx-indexed-db.service.ts

```diff
@@ -33,6 +33,41 @@ export class NgxIndexedDBService {
 		}
 	}
 
+	createObjectStores(
+		storeSchemas: ObjectStoreMeta[],
+		migrationFactory?: () => { [key: number]: (db: IDBDatabase, transaction: IDBTransaction) => void }
+	) {
+		if (migrationFactory != null) {
+			CreateObjectStore(
+				this.indexedDB,
+				this.dbConfig.name,
+				this.dbConfig.version,
+				storeSchemas,
+				migrationFactory
+			);
+		} else {
+			CreateObjectStore(this.indexedDB, this.dbConfig.name, this.dbConfig.version, storeSchemas);
+		}
+	}
+
+	createObjectStore(
```

Comment:
Duplicated method, you can make it work with only one of them. Keep it DRY

Reply:
Will delete this one. Looks redundant.